Dockerfile

```diff
@@ -14,12 +16,12 @@ WORKDIR /app
 
 RUN apt update
 RUN apt install -y sshpass python3.7 python3-pip qemu-utils wget nginx
-RUN if [ "$(uname -m)" == "x86_64" ]; then \
+RUN if [[ "$(uname -m)" == "x86_64" ]]; then \
```

Comment:
by default container build env is sh and it doesn't support [[ 
use below snippet
```
RUN uname -m && if [ "$(uname -m)" = "x86_64" ]; then \
        echo "Attempting to download azcopy for x86_64..." && \
        wget https://azcopyvnext.azureedge.net/release20201021/azcopy_linux_amd64_10.6.1.tar.gz && \
        tar -zxf ./azcopy_linux_amd64_10.6.1.tar.gz && \
        mv ./azcopy_linux_amd64_10.6.1/azcopy /usr/bin && \
        chmod +x /usr/bin/azcopy && \
        echo "azcopy installation for x86_64 succeeded." ; \
    elif [ "$(uname -m)" = "aarch64" ]; then \
        echo "Attempting to download azcopy for aarch64..." && \
        wget -O azcopy_linux_arm64_10.18.1.tar.gz https://aka.ms/downloadazcopy-v10-linux-arm64 && \
        tar -zxf ./azcopy_linux_arm64_10.18.1.tar.gz && \
        mv ./azcopy_linux_arm64_10.18.1/azcopy /usr/bin && \
        chmod +x /usr/bin/azcopy && \
        echo "azcopy installation for aarch64 succeeded." ; \
    else \
        echo "Unexpected architecture. azcopy was not installed." ; \
    fi